content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,78 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Kubernetes' implementation of dual-stack networking is in flux. You may well find that the design is changed in future releases.{{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simultaneously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
```

Comment:
```suggestion
Enabling IPv4/IPv6 dual stack on your Kubernetes cluster provides the following features:
```

Reply:
Does this feature improve cluster operators's experience if they want to deploy Pods with multiple IPv4 addresses?
If so, there's scope for documenting that as well. I think I would log an issue to make that (potential) work visible.

Reply:
That enhancement is not yet available. We still only support at most a single address block from each family (IPv4 or IPv6) We can readdress this once it is